src/Microsoft.TemplateSearch.TemplateDiscovery/Program.cs

```diff
@@ -141,6 +143,11 @@ private static RootCommand CreateCommand()
                 Description = $"Path to pre-downloaded packages. If specified, the packages won't be downloaded from NuGet.org.",
             }.ExistingOnly());
 
+            Option<string> latestSdkToTestOption = (new Option<string>("--latestVersionToTest")
```

Comment:
nit, probably do not address: not sure if this option worth the effort. 
Probably for latest version we just want to run the tests without global.json and use that version (whatever it is)

Reply:
I somewhat agree. It may help us if we run wrong dotnet installation containing only an older version though.